src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -7,185 +7,185 @@ partial class ApplicationEngine
-            [OpCode.PUSH8] = 30,
-            [OpCode.PUSH9] = 30,
-            [OpCode.PUSH10] = 30,
-            [OpCode.PUSH11] = 30,
-            [OpCode.PUSH12] = 30,
-            [OpCode.PUSH13] = 30,
-            [OpCode.PUSH14] = 30,
-            [OpCode.PUSH15] = 30,
-            [OpCode.PUSH16] = 30,
-            [OpCode.NOP] = 30,
-            [OpCode.JMP] = 70,
-            [OpCode.JMP_L] = 70,
-            [OpCode.JMPIF] = 70,
-            [OpCode.JMPIF_L] = 70,
-            [OpCode.JMPIFNOT] = 70,
-            [OpCode.JMPIFNOT_L] = 70,
-            [OpCode.JMPEQ] = 70,
-            [OpCode.JMPEQ_L] = 70,
-            [OpCode.JMPNE] = 70,
-            [OpCode.JMPNE_L] = 70,
-            [OpCode.JMPGT] = 70,
-            [OpCode.JMPGT_L] = 70,
-            [OpCode.JMPGE] = 70,
-            [OpCode.JMPGE_L] = 70,
-            [OpCode.JMPLT] = 70,
-            [OpCode.JMPLT_L] = 70,
-            [OpCode.JMPLE] = 70,
-            [OpCode.JMPLE_L] = 70,
-            [OpCode.CALL] = 22000,
-            [OpCode.CALL_L] = 22000,
-            [OpCode.CALLA] = 22000,
-            [OpCode.THROW] = 30,
-            [OpCode.THROWIF] = 30,
-            [OpCode.THROWIFNOT] = 30,
+            [OpCode.PUSHINT8] = 3,
+            [OpCode.PUSHINT16] = 3,
+            [OpCode.PUSHINT32] = 3,
+            [OpCode.PUSHINT64] = 3,
+            [OpCode.PUSHINT128] = 12,
+            [OpCode.PUSHINT256] = 12,
+            [OpCode.PUSHA] = 12,
+            [OpCode.PUSHNULL] = 3,
+            [OpCode.PUSHDATA1] = 18,
+            [OpCode.PUSHDATA2] = 130,
+            [OpCode.PUSHDATA4] = 1100,
+            [OpCode.PUSHM1] = 3,
+            [OpCode.PUSH0] = 3,
+            [OpCode.PUSH1] = 3,
+            [OpCode.PUSH2] = 3,
+            [OpCode.PUSH3] = 3,
+            [OpCode.PUSH4] = 3,
+            [OpCode.PUSH5] = 3,
+            [OpCode.PUSH6] = 3,
+            [OpCode.PUSH7] = 3,
+            [OpCode.PUSH8] = 3,
+            [OpCode.PUSH9] = 3,
+            [OpCode.PUSH10] = 3,
+            [OpCode.PUSH11] = 3,
+            [OpCode.PUSH12] = 3,
+            [OpCode.PUSH13] = 3,
+            [OpCode.PUSH14] = 3,
+            [OpCode.PUSH15] = 3,
+            [OpCode.PUSH16] = 3,
+            [OpCode.NOP] = 3,
+            [OpCode.JMP] = 7,
+            [OpCode.JMP_L] = 7,
+            [OpCode.JMPIF] = 7,
+            [OpCode.JMPIF_L] = 7,
+            [OpCode.JMPIFNOT] = 7,
+            [OpCode.JMPIFNOT_L] = 7,
+            [OpCode.JMPEQ] = 7,
+            [OpCode.JMPEQ_L] = 7,
+            [OpCode.JMPNE] = 7,
+            [OpCode.JMPNE_L] = 7,
+            [OpCode.JMPGT] = 7,
+            [OpCode.JMPGT_L] = 7,
+            [OpCode.JMPGE] = 7,
+            [OpCode.JMPGE_L] = 7,
+            [OpCode.JMPLT] = 7,
+            [OpCode.JMPLT_L] = 7,
+            [OpCode.JMPLE] = 7,
+            [OpCode.JMPLE_L] = 7,
+            [OpCode.CALL] = 220,
+            [OpCode.CALL_L] = 220,
+            [OpCode.CALLA] = 220,
+            [OpCode.THROW] = 3,
```

Comment:
THROWXX will need a readjustment after https://github.com/neo-project/neo-vm/pull/294

Reply:
ok, I will keep an eye on that.